deepchem/models/tensorgraph/layers.py

```diff
@@ -4303,3 +4303,29 @@ def batch_mat_mult(self, A, B):
     result = tf.matmul(A_reshape, B)
     result = tf.reshape(result, tf.stack([A_shape[0], A_shape[1], axis_2]))
     return result
+
+
+class Hingeloss():
+
+  def __init__(self, in_layers=None, weights, batch_size, labels, reg,
+               **kwargs):
+
+    self.weights = weights
+    self.batch_size = batch_size
+    self.labels = labels
+    self.reg = reg
+    super(Hingeloss, self).__init__(in_layers, **kwargs)
+
+  def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
+    inputs = self.get_input_tensors(in_layers)
```

Comment:
can you add documentation on the in layers?  It seems like batch_size is a tensor and not a scaler.  I am confused at what each of them should be.

Reply:
I didn't get what you are trying to ask, what am I supposed to add documentation for/

Reply:
I think I got it. Will do that in a few minutes.

Reply:
Just add docstrings for the class and for `__init__()`.  We're trying to be good about having documentation for all layers, though some of the older ones are missing it.

Reply:
ok, I added for the __init__() in the update and also add about the class